### ISSUE TITLE
contributor

### ISSUE THREAD

**Issue by softloud** [softloud/supermetroid#12]
Hi @jeremander, may I list you as a contributor on this packaged analysis? Thanks so much for your assistance with splitsio.

**Comment by jeremander** [comment]
Sure, much appreciated!  Looking at your EDA/graphs, I'm very impressed.  Good luck on your presentation!

**Comment by softloud** [comment]
Updated [README](https://softloud.github.io/supermetroid/index.html#open-python-speed-runner-api-implementations) with a description of your package and it warms my open science heart to have you [listed as a contributor](https://softloud.github.io/supermetroid/authors.html).

btw, I checked out your thesis; very closely aligned to [my own](https://link.springer.com/epdf/10.1007/s11083-017-9426-3?author_access_token=Tx3I0PNwg5dFrahH71DncPe4RwlQNchNByi7wbcMAY5xivQFAeypeZYbpkYiIm56ubi44Xeb3jQJh6Y2LL8MDMMBjFNWvxCIzyDBkmDI7ZtY6XFcn9qUBg5X3MB3WdDpzOfXDt4InQo_9PdfLHknPw==) (as you know, in mathematics, authorship is alphabetical). 

So nice to meet another algebraic nerd in the wilds of open data science!
